src/mem/MemCtrl.py

```diff
@@ -58,6 +58,8 @@ class MemCtrl(QoSMemCtrl):
     # bus in front of the controller for multiple ports
     port = ResponsePort("This port responds to memory requests")
 
+    dram_cache_size = Param.Int('1024', "DRAM cache size")
```

Comment:
1024 what? 

This param should probably be a "MemorySize"

Reply:
This is in Bytes.
Should it be '1024 B' or '1024'?